infra/bootstrapper/locals.tf

```diff
@@ -0,0 +1,50 @@
+# Review each value of this file and update it according to your needs.
+
+locals {
+  prefix          = "dx"
+  env_short       = "d"
+  location        = "italynorth"
+  domain          = "typescript"
+  instance_number = "01"
+
+  adgroups = {
+    admins_name = "io-p-adgroup-admin"
+    devs_name   = "io-p-adgroup-developers"
+  }
+
+  runner = {
+    cae_name                = "${local.prefix}-${local.env_short}-itn-github-runner-cae-01"
```

Comment:
question: can't we use the provider method for names?

Reply:
I don't see it really convenient in this case, as resources are not grouped under the same domain (`network`, `common`, `github` all require a specific value for `domain`). It can be done but the code would look messy and harder to read in my opinion